java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/Util.java

```diff
@@ -528,7 +528,7 @@ public static String typeIdToClass(String id) {
             if (buf.length() > 0) {
                 buf.append('.');
             }
-            buf.append(fixKwd(s));
```

Comment:
A result of this change is that if a user has a class/exception that collides with a Java keyword,
they will be stuck using 3.7. This is not optimal.

Reply:
> if a user has a class/exception that collides with a Java keyword

Nobody has such a class or exception, so this is definitely not a concern.